specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-01-preview/images.json

```diff
@@ -0,0 +1,348 @@
+{
+   "swagger": "2.0",
+   "info": {
+      "title": "Cognitive Services API",
+      "description": "Cognitive Services API to support OpenAI Dall-E",
+      "version": "2023-04-01-preview"
+   },
+   "host": "eastus.api.cognitive.microsoft.com",
+   "basePath": "/dalle",
+   "schemes": ["https"],
+   "securityDefinitions": {
+      "Api-Key": {
+        "type": "apiKey",
+        "name": "Api-Key",
+        "in": "header",
+        "description": "Provide your Cognitive Services Azure OpenAI account key here."
+      }
+   },
+   "security": [
+      {
+        "Api-Key": []
+      }
+    ],
+   "parameters": {
+      "apiVersionQueryParameter": {
+        "in": "query",
+        "name": "api-version",
+        "description": "The requested API version.",
+        "required": true,
+        "type": "string"
+      }
+   },
+   "definitions": {
+      "Status": {
+         "description": "The status of the operation.",
+         "type": "string",
+         "enum": [
```

Comment:
>enum": [ [](http://example.com/codeflow?start=10&length=8)

Use extensions for auto rest, e.g.

    "State": {
      "title": "State",
      "description": "The state of a job or item.",
      "enum": [
        "notRunning",
        "running",
        "succeeded",
        "canceled",
        "failed",
        "deleted"
      ],
      "type": "string",
      "readOnly": true,
      "x-ms-enum": {
        "name": "State",
        "modelAsString": true,
        "values": [
          {
            "value": "notRunning",
            "description": "The operation was created and is not queued to be processed in the future."
          },
          {
            "value": "running",
            "description": "The operation has started to be processed."
          },
          {
            "value": "succeeded",
            "description": "The operation has successfully be processed and is ready for consumption."
          },
          {
            "value": "canceled",
            "description": "The operation has been canceled and is incomplete."
          },
          {
            "value": "failed",
            "description": "The operation has completed processing with a failure and cannot be further consumed."
          },
          {
            "value": "deleted",
            "description": "The entity has been deleted but may still be referenced by other entities predating the deletion."
          }
        ]
      }
    },

Reply:
Used the format as you suggested. 
Is it an azure guidelines that enum values should not start with a capital letter?

Reply:
Fixed the format. Regaring the capital letters, following the guidelines from here https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#long-running-operations, the status would always start with a capital letter.